.pre-commit-config.yaml

```diff
@@ -30,13 +30,13 @@ repos:
                 libraries/AP_ADSB/AP_ADSB_Sagetech_MXS.h
               )$
         -   id: check-added-large-files
-        -   id: check-executables-have-shebangs
```

Comment:
Please stop suggesting we remove hooks. These are good additions. I would like to keep them.